cli/src/config/templates.toml

```diff
@@ -148,3 +140,12 @@ commit_summary_separator = 'label("separator", " | ")'
 'format_time_range(time_range)' = '''
   time_range.start().ago() ++ label("time", ", lasted ") ++ time_range.duration()'''
 'format_timestamp(timestamp)' = 'timestamp'
+
+# For internal use only
+__builtin_decorated_change_id = '''
+label(
```

Comment:
nit: maybe it's okay for user to override the `decorated_change_id` template?

It's fuzzy whether a template alias is for internal use or not. `description_placeholder` isn't meant to be customized by user, but I don't mind if it's overridden by user config.

Reply:
Either way is OK by me, I'll think a bit more if there's a reason to call this a "builtin" (I'd probably have waited regardless in case you wanted to look at the latest changes).

Reply:
I'll just change it to `builtin_change_id_with_hidden_and_divergent_info`, without extra warnings.